src/TLuaInterpreter.cpp

```diff
@@ -17197,6 +17198,110 @@ int TLuaInterpreter::getMouseEvents(lua_State * L)
     return 1;
 }
 
+int TLuaInterpreter::setConfig(lua_State * L)
+{
+    auto& host = getHostFromLua(L);
+    QString key = getVerifiedString(L, __func__, 1, "key");
+    if (key.isEmpty()) {
+        return warnArgumentValue(L, __func__, "you must provide key");
+    }
+
+    auto success = [&]()
+    {
+        if (mudlet::debugMode) {
+            TDebug(Qt::white, Qt::blue) << qsl("setConfig: a script has changed %1\n").arg(key) >> &host;
+        }
+        lua_pushboolean(L, true);
+        return 1;
+    };
+
+    if (host.mpMap && host.mpMap->mpMapper) {
+        if (key == qsl("roomSize")) {
+            host.mpMap->mpMapper->slot_setRoomSize(getVerifiedInt(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == qsl("exitSize")) {
+            host.mpMap->mpMapper->slot_setExitSize(getVerifiedInt(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == qsl("roundRooms")) {
+            host.mpMap->mpMapper->slot_toggleRoundRooms(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == qsl("showRoomIds")) {
+            host.mpMap->mpMapper->slot_setShowRoomIds(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == qsl("showMapInfo")) {
+            host.mMapInfoContributors.insert(getVerifiedString(L, __func__, 2, "value"));
+            host.mpMap->mpMapper->slot_updateInfoContributors();
+            return success();
+        }
+        if (key == qsl("hideMapInfo")) {
+            host.mMapInfoContributors.remove(getVerifiedString(L, __func__, 2, "value"));
+            host.mpMap->mpMapper->slot_updateInfoContributors();
+            return success();
+        }
+#if defined(INCLUDE_3DMAPPER)
+        if (key == qsl("show3dView")) {
+            host.mpMap->mpMapper->slot_toggle3DView(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+#endif
+        if (key == qsl("panelVisible")) {
+            host.mpMap->mpMapper->slot_setMapperPanelVisible(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == qsl("showRoomBorders")) {
+            host.mMapperShowRoomBorders = (getVerifiedBool(L, __func__, 2, "value"));
```

Comment:
You have some unneeded braces here.

```suggestion
            host.mMapperShowRoomBorders = getVerifiedBool(L, __func__, 2, "value");
```